inventories/group_vars/all

```diff
@@ -260,6 +260,16 @@ terrain:
   log_file: terrain-docker.log
   max_heap: "{{ max_heap.high }}"
 
+image_janitor:
+  host: "{% if use_overlay %}image-janitor{% else %}{{ groups['image-janitor'][0] }}{% endif %}"
+  port:
+  base: "http://{% if use_overlay %}image-janitor{% else %}{{ groups['image-janitor'][0] }}{% endif %}"
```

Comment:
(noting as well that a bunch of these things aren't used, but we probably need to go through stuff in general to deal with that problem, so it's not specific to this PR)